pkgs/development/compilers/ghc/7.10.1.nix

```diff
@@ -13,6 +13,13 @@ let
     ''}
   '';
 
+  # See: https://github.com/haskell/cabal/issues/2320
```

Comment:
I tried the following

        ghcjs = packages.ghc7101.callPackage ../development/compilers/ghcjs {
          ghc = compiler.ghc7101;
          Cabal = packages.ghc7101.Cabal_1_22_1_1;
        };

... but it looks like `ghcjs-pkg` is still using the Cabal library bundled with GHC. It appears that we'll just have to wait for the next GHC release.